src/concepts/pipeline/glean_data.md

```diff
@@ -43,21 +45,33 @@ browser:
 
 It would be available in the column `metrics.boolean.browser_is_default`.
 
-```sql
--- Count number of pings where Fenix is the default browser
-SELECT
-  COUNT(*),
-  COUNTIF(metrics.boolean.browser_is_default)
-FROM
-  -- We give the table an alias so that the table name `metrics` and field name
-  -- `metrics` don't conflict.
-  org_mozilla_fenix.metrics AS m
-WHERE
-  date(submission_timestamp) = '2019-11-11'
-```
```

Comment:
I think it's better to leave the examples in the cookbook section, and have the focus here be on being a reference.